applications/nrf_desktop/src/modules/Kconfig.ble

```diff
@@ -215,4 +215,56 @@ source "subsys/logging/Kconfig.template.log_config"
 
 endif
 
+comment "BLE QoS management"
+
+config DESKTOP_BLE_QOS_ENABLE
+	bool "Enable BLE channel map management"
+	depends on BT_CENTRAL
+	depends on BT_LL_NRFXLIB
+	select BT_HCI_VS_EVT_USER
+	help
+	  Enable device to avoid congested RF channels.
+
+config DESKTOP_BLE_QOS_INTERVAL
+	int "Processing interval for QoS thread [ms]"
+	default 1000
+	depends on DESKTOP_BLE_QOS_ENABLE
+	help
+	  Configure processing interval for QoS algorithm.
+	  Longer intervals means more time to accumulate CRC stats,
+	  and vice versa.
+
+config DESKTOP_BLE_QOS_STACK_SIZE
+	int "Base stack size for QoS thread"
+	default 240
```

Comment:
```
default 1024 if DESKTOP_BLE_QOS_STATS_PRINTOUT_ENABLE
default 240
```
then you can remove `DESKTOP_BLE_QOS_STATS_PRINT_STACK_SIZE` and simplify the code

Reply:
Thanks, I was not aware of this option

Reply:
:)

Reply:
Ref offline conversation, conditional default does not seem to adapt to changes made in menuconfig

Reply:
discussed offline - this indeed does not work if changes are done in menuconfig...